code/ship/ship.cpp

```diff
@@ -2665,6 +2666,9 @@ int parse_ship_values(ship_info* sip, bool first_time, bool replace)
 	//Parse the engine sound
 	parse_sound("$EngineSnd:", &sip->engine_snd, sip->name);
 
+	if(optional_string("$Minimum Engine Volume:"))
+		stuff_float(&sip->min_engine_vol);
```

Comment:
Actually I specifically chose not to validate it, in order to not unnecessarily prevent one from doing something weird and unusual with it. You could for example use values >1.0 to make the sound volume drop as speed increases, which could have unexpected uses for, say, an otherworldly alien probe.